packages/forms/frontend/sirius-components-forms/src/propertysections/ButtonPropertySection.tsx

```diff
@@ -11,8 +11,16 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 import { useMutation } from '@apollo/client';
-import { ServerContext, ServerContextValue, getCSSColor, useMultiToast } from '@eclipse-sirius/sirius-components-core';
+import {
+  ServerContext,
+  ServerContextValue,
+  getCSSColor,
+  theme,
```

Comment:
I somehow missed that you have an hardcoded reference to the Sirius Components base theme here. Use the current theme instead.